src/reconcile/rec_visibility.c

```diff
@@ -590,16 +591,7 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, W
         WT_ERR(__rec_update_save(session, r, ins, rip,
           upd_select->upd != NULL && upd_select->upd->type == WT_UPDATE_TOMBSTONE ? NULL :
                                                                                     upd_select->upd,
-          supd_restore, upd_memsize));
-        /*
-         * Mark the selected update (and potentially the tombstone preceding it) as being destined
-         * for the data store. Subsequent reconciliations should know that they can select this
-         * update regardless of visibility.
-         */
-        if (upd_select->upd != NULL)
-            F_SET(upd_select->upd, WT_UPDATE_DS);
-        if (tombstone != NULL)
-            F_SET(tombstone, WT_UPDATE_DS);
+          tombstone, supd_restore, upd_memsize));
         upd_saved = upd_select->upd_saved = true;
```

Comment:
Suppose we only select a tombstone as such there isn't a saved update chain, and the upd_select->upd = tombstone. Then should we set WT_UPDATE_DS if we don't return EBUSY in [rec_row.c ](https://github.com/wiredtiger/wiredtiger/blob/c83ffbb031d29265f92a55e1b4703b10fd79d85b/src/reconcile/rec_row.c#L931) for fixing up history store entries?

Reply:
Not required as explained by the comment in hs_rec.c.